types/node-insights/index.d.ts

```diff
@@ -22,6 +22,7 @@ declare class Insights {
     static collectorBaseURL: string;
     static queryBaseURL: string;
 
+    data: Array<Record<string, any>>;
```

Comment:
I think the intention of library was to make this a 'private' property since it doesn't go through `Object.defineProperty` explicitly like other properties.